lib/web_apis/release.es6.js

```diff
@@ -24,6 +57,22 @@ foam.CLASS({
     },
     {
       class: 'String',
+      name: 'friendlyBrowserVersion',
+      transient: true,
+      getter: function() {
+        const names = this.FRIENDLY_BROWSER_VERSIONS;
+        if (!names[this.browserName]) return this.browserVersion;
+        const versionNames = names[this.browserName];
+        const version = this.browserVersion.split('.');
```

Comment:
This works for the current state of data for Safari. Simple prefix matching might be more useful long-term, though I guess you said you wanted to do the right thing (deal with it as the data is gathered) so that long-term usage probably doesn't exist.